docs/standard/tour.md

```diff
@@ -14,137 +14,130 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
+* C# is simple, powerful, type-safe, and object-oriented, while retaining the expressiveness and elegance of C-style languages. Anyone familiar with C and similar languages finds few problems in adapting to C#. Check out the [C# Guide](../csharp/index.md) to learn more about C#.
 
-* F# is a cross-platform, functional-first programming language that also supports traditional object-oriented and imperative programming.  Check out the [F# Guide](../fsharp/index.md) to learn more about F#.
+* F# is a cross-platform, functional-first programming language that also supports traditional object-oriented and imperative programming. Check out the [F# Guide](../fsharp/index.md) to learn more about F#.
 
-* Visual Basic is an easy language to learn that you can use to build a variety of applications that run on .NET.
+* Visual Basic is an easy language to learn that you use to build a variety of apps that run on .NET. Among the .NET languages, the syntax of VB is the closest to ordinary human language often making it easier for people new to software development to pick up.
 
 ## Automatic memory management
 
-.NET uses [garbage collection](garbagecollection/index.md) to provide automatic memory management for programs.  The GC operates on a lazy approach to memory management, preferring application throughput to the immediate collection of memory.  To learn more about the .NET GC, check out [Fundamentals of garbage collection (GC)](garbagecollection/fundamentals.md).
+.NET uses [garbage collection (GC)](garbagecollection/index.md) to provide automatic memory management for programs. The GC operates on a lazy approach to memory management, preferring app throughput to the immediate collection of memory. To learn more about the .NET GC, check out [Fundamentals of garbage collection (GC)](garbagecollection/fundamentals.md).
 
 The following two lines both allocate memory:
 
 [!code-csharp[MemoryManagement](../../samples/csharp/snippets/tour/MemoryManagement.csx#L1-L2)]
 
 There is no analogous keyword to de-allocate memory, as de-allocation happens automatically when the garbage collector reclaims the memory through its scheduled run.
 
-The garbage collector is just one of the services that help ensure *memory safety*.  The invariant of memory safety is very simple: a program is memory safe if it accesses only memory that has been allocated (and not freed).  For instance, the runtime ensures that programs do not index off the end of an array or access a phantom field off the end of an object.
+The garbage collector is just one of the services that help ensure *memory safety*. The invariant of memory safety is very simple: a program is memory safe if it accesses only allocated (and not freed) memory. For instance, the runtime ensures that programs don't index off the end of an array or access a phantom field off the end of an object.
```

Comment:
This paragraph could use some work:
1. Eliminate just in "just one of the services". It begs the question of what the others are and sounds like marketing hype.
2. I have no idea of what "the invariant of memory safety" is. I'd eliminate the sentence, and just begin "A program is memory safe..."
3. The dichotomy is not allocated and freed memory; it's allocated and unallocated memory. So delete "(and not freed)"
4. The last sentence over-explains in a rather mystifying way. Better: "For instance, the runtime ensures that apps don't access unallocated memory beyond the bounds of an array."